queries/javascript/highlights.scm

```diff
@@ -1,23 +1,37 @@
 ; Types
 
 ; Javascript
+
+; Properties
+;-----------
+
+(property_identifier) @property
+
 ; Special identifiers
 ;--------------------
 
+(identifier) @variable
+
 ((identifier) @constant
- (#match? @constant "^[A-Z_][A-Z\\d_]+$"))
+ (match? @constant "^[A-Z_][A-Z\\d_]+$"))
```

Comment:
Is this something new in treesitter where predicates no longer start with #?

Reply:
I believe it's deprecated... both are valid though

Reply:
Yeah, they updated the syntax. The new version is without #

Reply:
Looks like this was the other way around? https://github.com/tree-sitter/tree-sitter/pull/615

> To disambiguate predicate expressions for named nodes, predicate names now must begin with #. I think this makes patterns with predicates much more readable, and it simplifies the parser.

If that's the case we should use the syntax with `#`

Reply:
Oh, then it was the other way around :smile:. It's not like we just try out what works.

Reply:
So we should keep the `#` then?

Reply:
we had this issue #76 where everyone replaced the stuff with `#` but then everyone preferred the version without `#`

Reply:
> So we should keep the # then?

Yeah, the other version isn't documented anymore, and using `#` is more explicit since it could collide with a named node (that's why they changed the syntax).